src/components/OffsiteLink.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+interface OffsiteLinkProps {
+  /**
+   * Destination URL.
+   */
+  to: string;
+  /**
+   * If true, will open the destination in a new window/tab
+   * and will add the `rel="noopener noreferrer"` guardrails.
+   */
+  targetBlank?: boolean;
+  /**
+   * Visible contents of the link.
+   * If omitted, will default to the destination URL (`to`).
+   */
+  children?: React.ReactNode;
+  className?: string;
+}
+
+/**
+ * Component for linking off-site.
+ * Defaults to opening links in a new window/tab (which can be disabled) and
```

Comment:
I'm going to push back a bit on this, too!

Why do we want links to open in a new tab? Users have the ability to choose whether to open in a new window/tab in every modern browser. If they click an external link and decide they wanted it to be in a new tab after all, they can hit the back button and middle click. If we force it to open in a new tab, users can't make that decision.

Reply:
This is where I appreciate your rethinking my priors.

I philosophically agree with you but have lost this battle so often that I not reflexively default to what I expect the client will want ("offsite links open in new tabs"). I am _ecstatic_ to invert this behavior and default to web-native behavior. Excellent pushback.